controllers/nstemplateset/features.go

```diff
@@ -0,0 +1,30 @@
+package nstemplateset
+
+import (
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"strings"
+)
+
+// shouldCreate checks if the object has a feature toggle annotation. If it does then check if the corresponding
+// feature is referenced in the NSTemplateSet feature annotation. Returns true if yes. It means this feature
+// should be enabled and the object should be created. It also returns true if the object doesn't have a feature annotation at all
+// which means it's a regular object, and it's not managed by any feature toggle and should be always created.
+// Otherwise, returns false.
+func shouldCreate(toCreate runtimeclient.Object, nsTmplSet *toolchainv1alpha1.NSTemplateSet) bool {
+	feature, found := toCreate.GetAnnotations()[toolchainv1alpha1.FeatureToggleNameAnnotationKey]
+	if !found {
+		return true // This object is a regular object and not managed by a feature toggle. Always create it.
+	}
+	// This object represents a feature. Let's check if this feature is among winners in the NSTemplateSet.
+	winners, found := nsTmplSet.GetAnnotations()[toolchainv1alpha1.FeatureToggleNameAnnotationKey]
+	if !found {
```

Comment:
I think `string.Split("", ",")` should really return an empty slice instead of confusing `[""]` :)
But anyway. Addressed here: https://github.com/codeready-toolchain/member-operator/pull/580/commits/48c03389ee23952d6d4ceb9ebc8b4f2dafda8693